packages/spectral/src/types/ActionPerformFunction.ts

```diff
@@ -72,4 +103,8 @@ export type ActionContext<
   flow: FlowAttributes;
   /** The time in UTC that execution started. */
   startedAt: string;
+  /** @TODO: Description */
+  invokeFlow: FlowInvoker<TFlows>;
+  /** @TODO: Description */
```

Comment:
Let's go with `Reference to the current execution and, when applicable, the current step.`